docs/ibexa_products/headless.md

```diff
@@ -0,0 +1,180 @@
+  The Back Office itself is such a channel.
+- Multi-repository, the same platform can use separate databases if impermeability is needed between channel groups.
+
+### Advanced features
+
+On top of this strong core, [[= product_name =]] Headless edition brings tools to increase user experience, from final front users to Back Office contributors.
+
+#### [[= product_name_headless =]] is a complete Digital Experience Platform (DXP)
+
+- [[= product_name_headless =]] comes with a personalization engine, allowing to recommend content to the front user according to its behavior,
+  or, when authenticated, contents matching user's segment/group. See [Personalization product guide](personalization_brochure.md) for more.
+- [[= product_name_headless =]]' scheduler allows to establish the future of the content and having a living front application with events even when the editorial team is absent or reduced. At midnight, during weekends or vacations, front users can discover new contents. A calendar summaries those scheduled content event. The calendar is extendable as everything in the Back Office, some event source could be added to coordinate content events with other company events.
+
+#### [[= product_name_headless =]] adds more ways to structure and organize your content
+
+- [[= product_name_headless =]] has specific feature to organize complex products and their catalogs, making it a strong Product Information Management (PIM) software.
+    - Product are organized per product types × variants × catalogs × categories × tags.
+    - Product attributes are grouped and factorized among product types. For example, fabric + color + size can be shared by many clothing product types.
+    - Product variants can rapidly be created by the automatic declination of attributes having a defined set of values.
+    - See [PIM product guide](pim_guide.md) for more.
+- [[= product_name_headless =]]' taxonomy feature allows to tag content items to organize them by topics in a much intuitive way for the editor than a content tree with multiple locations would.
+  Tags themselves are organized in a tree, and synonyms are linked to a favorite terms. This tags organisation can be the task of a supervisor who won't need to move content items around a corporate content tree. At search time, tags can be keywords with a high value in relevance score to help the end user having results closer to the searched topic.
+
+#### [[= product_name_headless =]] is a collaborative workplace
+
+- Version comparison helps to track changes and to solve concurrent editing conflicts.
+- Workflows helps with collaborative editing chain.
+  A built-in “Quick review“ workflow allows an editor to send a content draft to a colleague for review, and comment or publishing.
+  But, as a framework, more complex workflows can be imagined, with several steps and paths, even some automated tasks.
+
+#### [[= product_name_headless =]] accelerates content editing
+
```

Comment:
I'd gonsider getting rid of the bullets here and turning this sectionit into a regular paragraph

Reply:
I didn't find how to address this properly.